### PR TITLE
Ports the saxophone from goonstation.

### PR DESCRIPTION
**Pull request by Spacedong**
Sprite:

https://vgy.me/TfiE1y.png

Gameplay: 

https://www.youtube.com/watch?v=GaoLU6zKaws

### CODE REVIEW

File: code/game/objects/items/weapons/clown_items.dm [hunk 1]
```diff
@@ -128,6 +128,26 @@
...
+	item_state = "sax"
+	force = 10
+	cooldowntime = 150
+	var/list/sounds = list('sound/items/sax.ogg', 'sound/items/sax2.ogg','sound/items/sax3.ogg','sound/items/sax4.ogg','sound/items/sax5.ogg')
+
+/obj/item/weapon/bikehorn/saxophone/attack_self(mob/user)
+	if(!spam_flag)
+		spam_flag = 1
+		playsound(src.loc, pick(sounds), 50, 1)
+		user.visible_message("<B>[user]</B> lays down a [pick("sexy", "sensuous", "libidinous","spicy","flirtatious","salacious","sizzling","carnal","hedonistic")] riff on \his saxophone!")
```
Comment (JohnGinnane) [inline code/game/objects/items/weapons/clown_items.dm:145]: You could add these words to a list pretty much the same way you have the sounds

Comment (Spacedong) [inline code/game/objects/items/weapons/clown_items.dm]: no thanks

Comment (TDcoolguy300) [reply code/game/objects/items/weapons/clown_items.dm:145]: Do it

Comment (Spacedong) [inline code/game/objects/items/weapons/clown_items.dm]: no

Comment (TheOneAndOnlyCreeperJoe) [reply code/game/objects/items/weapons/clown_items.dm:145]: you've got no choice.

**Comment by McBawbaggings** [comment]
Failed compile.
TEST
YOUR
CODE

**Comment by SpankMasterson** [comment]
+1 contingent on spacedong writing working code

**Comment by asskek** [comment]
FUCK THIS BRING BACK TREKCHEM

**Comment by Spacedong** [comment]
@Hackole @McBawbaggings this works, thots?

**Comment by Hackole** [comment]
Sure. +1

**Comment by Spacedong** [comment]
this is ready for a merge

**Comment by Spacedong** [comment]
@McBawbaggings MERGE THIIS ITS READY

**Comment by JohnGinnane** [comment]
Code's good, head's happy, merging

**Pull request merged**